thoth/python/source.py

```diff
@@ -375,13 +337,32 @@ def provides_package_version(self, package_name: str, package_version: str) -> b
             # Package was not found on the index.
             return False
 
+    @lru_cache(maxsize=10)
+    def get_package_data(self, package_name: str, package_version: str, with_included_files: bool = False) -> list:
```

Comment:
It will be a good idea to move `get_package_data` and `get_package_hashes` to `Artifact` object as it semantically corresponds there.

Reply:
This is hard because these functions handle more than one artifact